skaffold.yaml

```diff
@@ -59,6 +59,10 @@ deploy:
         repo: https://openfga.github.io/helm-charts
         valuesFiles: ["deployments/helm/openfga/values.yaml"]
         wait: false
+      - name: mailhog
```

Comment:
question: why `mailhog` and not the `maildev` used in the original PR?

Reply:
`maildev` does not maintain the helm chart well.